CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## Unversioned
 
+- Major: Added clip creation support. You can create clips with `/clip` command, `Alt+X` keybind or `Create a clip` option in split header's context menu. (#2271)
```

Comment:
```suggestion
- Major: Added clip creation support. You can create clips with `/clip` command, `Alt+X` keybind or `Create a clip` option in split header's context menu. [This requires a new scope so re-authentication will be required to use it.] (#2271)
```
moved the 'new scope' information to the changelog instead

Reply:
yeah I forgot to include that in the changelog entry, good catch :+1: